contracts/cw-proposal-single/src/contract.rs

```diff
@@ -155,17 +179,6 @@ pub fn execute_propose(
         return Err(ContractError::InactiveDao {});
     }
 
-    // Check that the sender is a member of the governance contract.
-    let sender_power = get_voting_power(
-        deps.as_ref(),
-        sender.clone(),
-        config.dao.clone(),
-        Some(env.block.height),
-    )?;
-    if sender_power.is_zero() {
-        return Err(ContractError::Unauthorized {});
-    }
```

Comment:
I think we need this check? Many DAOs only want members to be able to submit proposals.

Like I don't want people submitting props on my multisigs....

Reply:
Maybe make a comment of where this check happens?

https://github.com/DA0-DA0/dao-contracts/blob/zeke/deposit-modules/packages/cw-pre-propose-base/src/execute.rs#L132